assets/protos/greet.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+option csharp_namespace = "Skyra.Grpc.Services";
+
+package greet;
+
+// The greeting service definition.
+service Greeter {
```

Comment:
do we need the greeter service? it's an easy way to test gRPC connections, but on the other hand, some of the other endpoints (such as adding points) are really easy to test too (via bloomrpc or some other testing client)